lib/puppet-strings/markdown/templates/data_type.erb

```diff
@@ -0,0 +1,78 @@
+### <%= name %>
+
+<% if text -%>
+<%= text %>
+<% elsif summary -%>
+<%= summary %>
+<% else -%>
+<%= "The #{name} class." %>
```

Comment:
Should this default refer to a "data type" instead of a "class"?

Reply:
Fixed